### PR TITLE
Added tooltip, fixed the height of tree and more small changes

### PR DESCRIPTION
**Pull request by AdimAnyvision**
### Checklist
- [ ] Does this need a new/update storybook example?
- [ ] Does this need new/update tests?

- [ ] run eslint
- [ ] run test
- [ ] run build

If your answer is yes to any of these, please make sure to include it in your PR.

> NOTE: Please submit all PRs to the `development` branch.

### CODE REVIEW

File: src/components/Tree/Tree.module.scss [hunk 1]
```diff
@@ -48,6 +56,7 @@
     align-items: center;
     margin: av-space() 0;
     padding: 0 av-space();
+    height: av-size(sz-40);
```
Comment (AdimAnyvision) [inline src/components/Tree/Tree.module.scss:59]: Add variable

File: src/components/Tree/Tree.module.scss [hunk 2]
```diff
@@ -83,6 +92,7 @@
     flex: 1;
     flex-direction: column;
     min-height: av-size(sz-480);
+    max-height: calc(100% - 2 * #{av-size(sz-48)});
```
Comment (AdimAnyvision) [inline src/components/Tree/Tree.module.scss:95]: Add variable

File: src/components/Tree/useFlattenTreeData/useFlattenTreeData.js [hunk 3]
```diff
@@ -128,13 +128,18 @@ const useFlattenTreeData = ({
...
 
   const handleSetSelectedNodesFromKeysArr = useCallback(
-    selectedKeysArr => {
+    (keysToAdd, keysToRemove = []) => {
       if (!Object.keys(flattenedNodes).length) return
-      selectedKeysArr.forEach(
+      keysToRemove.forEach(
+        key =>
+          (flattenedNodes[key] = { ...flattenedNodes[key], isSelected: false }),
+      )
```
Comment (BenTAnyvision) [inline src/components/Tree/useFlattenTreeData/useFlattenTreeData.js:136]: Here you could use `reduce`,
To keep `flattenedNodes` immutable.
(If you want)
Like this:
```suggestion
    const newFlattenedNodes = keysToRemove.reduce((newFlattenedNodes, key) => (
      { ...newFlattenedNodes[key], isSelected: false }
    ), flattenedNodes)
```

Comment (BenTAnyvision) [reply src/components/Tree/useFlattenTreeData/useFlattenTreeData.js:136]: Many functions are mutating the global state. So for this to work the whole thing will need to be refactored. So never mind for now.

**Review by BenTAnyvision — APPROVED** [review]

**Pull request merged**